### DISCUSSION TITLE
Auth UI Theme Supa  color “brand” does not work in production.

### DISCUSSION THREAD

**Discussion by gclcodeguy** [supabase/supabase#16909] [Contribute to Supabase]
# Bug report

<!-- 

I confirm this is a bug with Supabase, not with my own application.  

## Describe the bug

When using Auth UI Theme Supa and possible any theme, the “brand” color does not work in production. This means the bg color of the sign in / sign up button is white, so with white text you cannot see the button text. 

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

It works in local host but does NOT WORK IN PRODUCTION on edge, chrome or safari deployed with netlify.

I worked around by changing brandButtonText to black but it does seem to be a bug. 

To reproduce just create an Auth UI with theme supa and deploy to production using netlify. 

Workaround below:

<Auth
            supabaseClient={supabase}
            view="sign_up"
            appearance={{
              theme: ThemeSupa,
              variables: {
                default: {
                  colors: {
                    brand: "hsl(153 60.0% 53.0%)",  //this does not display in the button background
                    brandAccent: "hsl(154 54.8% 45.1%)",
                    brandButtonText: "black",
                    defaultButtonBackground: "hsl(153 60.0% 53.0%)",
                    defaultButtonBackgroundHover: "#eaeaea",
                    defaultButtonBorder: "lightgray",
                    defaultButtonText: "black",
                    dividerBackground: "#eaeaea",
                    inputBackground: "transparent",
                    inputBorder: "lightgray",
                    inputBorderHover: "gray",
                    inputBorderFocus: "gray",
                    inputText: "black",
                    inputLabelText: "gray",
                    inputPlaceholder: "darkgray",
                    messageText: "gray",
                    messageTextDanger: "red",
                    anchorTextColor: "gray",
                    anchorTextHoverColor: "darkgray",
                  },
                },
              },
            }}
            theme="auto"
            providers={[]}
          />

## Expected behavior

Expect brandButton color to determine the color of the button.

## Screenshots
![IMG_8362](https://github.com/supabase/supabase/assets/108017801/ba4e794b-4a6e-4d11-ae45-1b62a88355ec)

## System information

- Windows
- chrome, safari, edge

## Additional context

Sign in buttons look fine in local and stupid in production.

**Comment by MildTomato** [comment]
Got any more details on this?

**Comment by gclcodeguy** [comment]
Also, I think there is a bug with the Auth UI whereby the sign in / sign up button does not show the background color intended by the brand – so you can get a white bg and white text meaning you can’t see the text. It works in local host but does NOT WORK IN PRODUCTION on edge, chrome or safari deployed with netlify.

This occurs with the ThemeSupa

I worked around by changing brandButtonText to black but it does seem to be a bug. Workaround below:

<Auth
            supabaseClient={supabase}
            view="sign_up"
            appearance={{
              theme: ThemeSupa,
              variables: {
                default: {
                  colors: {
                    brand: "hsl(153 60.0% 53.0%)",  //this does not display in the button background
                    brandAccent: "hsl(154 54.8% 45.1%)",
                    brandButtonText: "black",
                    defaultButtonBackground: "hsl(153 60.0% 53.0%)",
                    defaultButtonBackgroundHover: "#eaeaea",
                    defaultButtonBorder: "lightgray",
                    defaultButtonText: "black",
                    dividerBackground: "#eaeaea",
                    inputBackground: "transparent",
                    inputBorder: "lightgray",
                    inputBorderHover: "gray",
                    inputBorderFocus: "gray",
                    inputText: "black",
                    inputLabelText: "gray",
                    inputPlaceholder: "darkgray",
                    messageText: "gray",
                    messageTextDanger: "red",
                    anchorTextColor: "gray",
                    anchorTextHoverColor: "darkgray",
                  },
                },
              },
            }}
            theme="auto"
            providers={[]}
          />

**Comment by gclcodeguy** [comment]
You can see my workaround with the white bg sign in button at:
https://golflists.com/register/
